src/common/validate/parameter-type/plural-type.js

```diff
@@ -0,0 +1,14 @@
+define([
+	"../parameter-type"
+], function( validateParameterType ) {
+
+return function( value, name ) {
+	validateParameterType(
+		value,
+		name,
+		value === undefined || value === "cardinal" || value === "ordinal",
+		"cardinal or ordinal"
```

Comment:
Please, let's make it a little more explicit with `String "cardinal" or "ordinal"`.